src/main/java/com/twilio/jwt/accesstoken/TaskRouterGrant.java

```diff
@@ -0,0 +1,73 @@
+package com.twilio.jwt.accesstoken;
+
+/**
+ * Grant used to access Twilio TaskRouter.
+ *
+ * <p>
+ *     For more information see:
+ *     <a href="https://www.twilio.com/docs/api/rest/access-tokens">
+ *         https://www.twilio.com/docs/api/rest/access-tokens
+ *     </a>
+ * </p>
+ */
+public class TaskRouterGrant implements Grant {
+
+    private String workspaceSid;
+    private String workerSid;
+    private String role;
```

Comment:
The role defines which TaskRouter endpoints are allowed.

Currently we have just a Worker JS SDK, who upon initialization specifies in the grant a role="worker". 

Our future Workspace SDK will have the role="workspace" who has different endpoints than the role="worker".